src/toil/cwl/cwltoil.py

```diff
@@ -3381,11 +3388,13 @@ def main(args: Union[List[str]] = None, stdout: TextIO = sys.stdout) -> int:
                 # were required.
                 rm_unprocessed_secondary_files(param_value)
 
+            logger.debug('tool ', tool)
             try:
                 wf1, _ = makeJob(
                     tool=tool,
                     jobobj={},
                     runtime_context=runtime_context,
+                    workflow_name='workflow',
```

Comment:
I'm not sure if I can get the workflow name from the ID at this step.

Reply:
How about `tool.tool['id']`  ?